src/cmd-koji-upload

```diff
@@ -195,20 +196,41 @@ class Build(_Build):
 
         return fname, True
 
-    def get_rpm_list(self, host=None):
+    def get_rpm_list(self, **kwargs):
```

Comment:
Minor: I think instead of a `kwargs` here, an enum would be cleaner. See e.g. https://github.com/coreos/coreos-assembler/blob/65c8e24c458e8b06d26debbf1d4e4c62f75ea2e7/src/cmd-diff#L31-L34 for an example.

So it'd be

```python
class RpmListKind(IntEnum):
    HOST = 0
    COMMITMETA = 1
    ...

def get_rpm_list(self, kind):
    if kind == HOST:
        ...
    ...
```

Fine with this too though!